### PR TITLE
milestone-2

### PR DESCRIPTION
**Pull request by cyonii**
In this milestone I have initialized the Redux actions, reducers and store. 

Additionally, I defined React components separating presentation and container components.

### CODE REVIEW

File: src/reducers/books.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actions/types';
+
+const initialState = [
+  { id: 'B34KS7K1T1T', title: 'Into the Wild', category: 'Biography' },
```
Comment (bellom) [inline src/reducers/books.js]: ### [Optional]
Let's simplify the code.
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'Into the Wild', category: 'Biography'}
```

File: src/reducers/books.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actions/types';
+
+const initialState = [
```
Comment (bellom) [inline src/reducers/books.js:6]: According to the instructions on pathwright, I think you need to move this to `src/index.js`

image

[Pathwright](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376028/)

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @cyonii  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check my comments under this review;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

**Review by rukundoeric — APPROVED** [review]
Hi @cyonii,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**